build/common.ps1

```diff
@@ -564,10 +564,13 @@ Function Test-ClientsProjects {
         [string]$Configuration = $DefaultConfiguration,
         [string]$MSBuildVersion = $DefaultMSBuildVersion
     )
+    
+    # We don't run command line tests on Dev15 as we don't build a nuget.exe for this version
     $testProjectsLocation = Join-Path $NuGetClientRoot test\NuGet.Clients.Tests
     $testProjects = Get-ChildItem $testProjectsLocation -Recurse -Filter '*.csproj' |
         %{ $_.FullName } |
-        ?{ -not $_.EndsWith('WebAppTest.csproj') }
+        ?{ -not $_.EndsWith('WebAppTest.csproj') -and 
+           -not ($_.EndsWith('NuGet.CommandLine.Test.csproj') -and ($MSBuildVersion -eq '15'))}
```

Comment:
We don't have a command line exe built with dev15 (look in build.ps1--we only ILMerge the Dev14 bits), so these tests are irrelevant to dev15.